packages/astro/package.json

```diff
@@ -63,9 +63,9 @@
   "dependencies": {
     "@astrojs/compiler": "^0.13.0",
     "@astrojs/language-server": "^0.8.10",
-    "@astrojs/markdown-remark": "^0.7.0-next.0",
-    "@astrojs/prism": "0.4.0",
-    "@astrojs/webapi": "^0.11.0",
+    "@astrojs/markdown-remark": "workspace:^0.7.0-next.0",
+    "@astrojs/prism": "workspace:^0.4.0",
+    "@astrojs/webapi": "workspace:^0.11.0",
```

Comment:
These `workspace:*` specifiers seemed to have snuck in here...

Reply:
I don't know if this was expected, but when I did `pnpm install`, this happened inside the lockfile:

```diff
- '@astrojs/markdown-remark': 0.7.0-next.0
- '@astrojs/prism': 0.4.0
- '@astrojs/webapi': 0.11.0
+ '@astrojs/markdown-remark': link:../markdown/remark
+ '@astrojs/prism': link:../astro-prism
+ '@astrojs/webapi': link:../webapi
```

That's why I added them

Reply:
Hmm, I wonder if you did the install from `packages/astro` rather than the root? Our `.npmrc` file has the following, which should prevent this.

```toml
# Important! Never install `astro` even when new version is in registry
prefer-workspace-packages=true
link-workspace-packages=true
save-workspace-protocol=false # This prevents the examples to have the `workspace:` prefix
```

Reply:
> Hmm, I wonder if you did the install from packages/astro rather than the root?

That was the problem. I pushed a fix